### PR TITLE
Fix permissions for scoped role member APIs

### PR DESCRIPTION
**Pull request by VinceSmith**

### CODE REVIEW

File: api-reference/beta/api/administrativeunit-delete-scopedrolemembers.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | AdministrativeUnit.ReadWrite.All, Directory.AccessAsUser.All    |
+|Delegated (work or school account) | RoleManagement.ReadWrite.Directory, Directory.AccessAsUser.All    |
 |Delegated (personal Microsoft account) | Not supported.    |
-|Application | AdministrativeUnit.ReadWrite.All |
+|Application | RoleManagement.ReadWrite.Directory |
```
Comment (dkershaw10) [inline api-reference/beta/api/administrativeunit-delete-scopedrolemembers.md]: Have you checked this with the engineers?  RoleManagement permissions for this operation seems a better fit, and more correct.

File: api-reference/beta/api/administrativeunit-get-scopedrolemembers.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | AdministrativeUnit.Read.All, AdministrativeUnit.ReadWrite.All, Directory.AccessAsUser.All    |
+|Delegated (work or school account) | Directory.Read.All, Directory.ReadWrite.All, Directory.AccessAsUser.All    |
```
Comment (dkershaw10) [inline api-reference/beta/api/administrativeunit-get-scopedrolemembers.md]: Wouldn't RoleManagement.Read.Directory be the least privileged permission here?

File: api-reference/beta/api/administrativeunit-post-scopedrolemembers.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | AdministrativeUnit.ReadWrite.All, Directory.AccessAsUser.All    |
+|Delegated (work or school account) | Directory.Read.All, Directory.ReadWrite.All, Directory.AccessAsUser.All    |
```
Comment (dkershaw10) [inline api-reference/beta/api/administrativeunit-post-scopedrolemembers.md]: ?
Directory.Read.All -> RoleManagement.ReadWrite.Directory (same for app only)

File: api-reference/beta/api/administrativeunit-delete-scopedrolemembers.md (hunk 1 above)
Comment (VinceSmith) [reply api-reference/beta/api/administrativeunit-delete-scopedrolemembers.md]: Already fixed. Not sure you're looking at the right version/branch?

File: api-reference/beta/api/administrativeunit-get-scopedrolemembers.md (hunk 2 above)
Comment (VinceSmith) [reply api-reference/beta/api/administrativeunit-get-scopedrolemembers.md]: Already fixed. Not sure you're looking at the right version/branch?

File: api-reference/beta/api/administrativeunit-post-scopedrolemembers.md (hunk 3 above)
Comment (VinceSmith) [reply api-reference/beta/api/administrativeunit-post-scopedrolemembers.md]: Good catch. Fixed.

**Pull request merged**